### ISSUE TITLE
Create a threshold flag, so the download stops before fill the entire disk.

### ISSUE THREAD

**Issue by rafaelbiriba** [rafaelbiriba/my-local-putio#1]
WE could introduce a `--disk-threshold` to stop the download if the disk space if lower than the threshold value.

This will avoid the script to fill the entire disk and cause OS errors.

**Comment by rafaelbiriba** [comment]
Useful line to return the empty space on a dir `df -Pk . | awk 'NR==2 {print $4}'`

**Comment by rafaelbiriba** [comment]
`get("files/#{id}")` to fetch the file stats. Field is size!
Example: 
```
{"file"=>{"content_type"=>"video/x-matroska", "crc32"=>"abc", "created_at"=>"2020-01-06T16:28:56", "extension"=>"mkv", "file_type"=>"VIDEO", "first_accessed_at"=>"2020-01-08T16:02:57", "folder_type"=>"REGULAR", "icon"=>"url.jpg", "id"=>123456, "is_hidden"=>false, "is_mp4_available"=>false, "is_shared"=>false, "name"=>"file-name.mkv", "opensubtitles_hash"=>"123", "parent_id"=>456, "screenshot"=>"url.jpg", "size"=>11239031017, "start_from"=>0, "updated_at"=>"2020-01-06T16:28:56"}, "status"=>"OK"}
```

**Comment by rafaelbiriba** [comment]
Merged with e53cc80e30de686e140c373c5b6291f570b1d5e8. 
Released as 4.3.0
